carl/envs/brax/carl_halfcheetah.py

```diff
@@ -50,4 +55,13 @@ def get_context_features() -> dict[str, ContextFeature]:
             "mass_ffoot": UniformFloatContextFeature(
                 "mass_ffoot", lower=1e-6, upper=np.inf, default_value=0.8845188
             ),
+            "target_distance": UniformFloatContextFeature(
```

Comment:
default distance of 0? == no goals?

Reply:
Put 100 everywhere now. That made me notice something else, though: right now by default the envs will be goal envs, I think. We should probably change that...

Reply:
yesss